packages/rocketchat-ui/views/app/room.coffee

```diff
@@ -493,7 +493,40 @@ Template.room.onCreated ->
 	@autorun =>
 		@subscribe 'fullUserData', Session.get('showUserInfo'), 1
 
-	@subscribe 'roomModerators', @data._id
+	Meteor.call 'getRoomModeratorsAndOwners', @data._id, (error, results) ->
+		if error
+			return toastr.error error.reason
+
+		for record in results
+			RoomModeratorsAndOwners.insert record
```

Comment:
This code will throw an exception if the room expires and open again, will insert duplicated IDs, you could user upsert here.